pay-from-browser/components/OrderTotal.tsx

```diff
@@ -1,14 +1,17 @@
-import { useCartState } from "../util/useCartState"
+import { StoreItemProps, useCartState } from "../util/useCartState"
 
-const OrderTotal = () => {
-    const items = useCartState().items
+export const totalPrice = (items: StoreItemProps[]) => {
+  return items.map(x => x.price).reduce((acc, cur) => acc + cur, 0)
```

Comment:
This could help to understand easier the code.

```suggestion
  return items.map(item => item.price).reduce((total, price) => total + price, 0)
```